notebooks/basic_training_notebook.ipynb

```diff
@@ -53,9 +60,7 @@
     "    !wget -O piper-sample-generator/models/en_US-libritts_r-medium.pt 'https://github.com/rhasspy/piper-sample-generator/releases/download/v2.0.0/en_US-libritts_r-medium.pt'\n",
```

Comment:
``!git clone -b mps-support https://github.com/kahrendt/piper-sample-generator`` will install a version that takes advantage of Apple silicon for generating the samples faster